MateriALL.gs

```diff
@@ -12,19 +12,25 @@ function onInstall(e) {
 
 // open the sidebar
 function showSidebar() {
-  const ui = HtmlService.createHtmlOutputFromFile('landing')
-      .setTitle('MateriALL');
+  const template = HtmlService.createTemplateFromFile('landing');
+  const ui = template.evaluate().setTitle('MateriALL');
```

Comment:
i had to make a few changes to this file so that i could use a global stylesheet that i could reuse across different files. i don't think it broke anything, but you should double check!